src/pdfjsWrapper.js

```diff
@@ -199,7 +199,7 @@ export default function(PDFJS) {
 
 			var pageRotate = (pdfPage.rotate === undefined ? 0 : pdfPage.rotate) + (rotate === undefined ? 0 : rotate);
 
-			var scale = canvasElt.offsetWidth / pdfPage.getViewport({ scale: 1 }).width * (window.devicePixelRatio || 1);
+			var scale = canvasElt.offsetWidth / pdfPage.getViewport({ scale: 1 }).width * 1;
```

Comment:
It's probably not going to be merged as the owner doesn't seem to be active on this package.

However, there is no point to multiply by 1, if `window.devicePixelRatio` is creating problems you might just remove the multiplication.

so instead of * 1 you just write:
```js
var scale = canvasElt.offsetWidth / pdfPage.getViewport({ scale: 1 }).width
```

Reply:
Thanks for your input. I did try this originally and it didn't always fix the issue in some edge cases. 1 is the second condition, it multiplies by window.devicePixelRatio if the relevant condition is met.